Dockerfile.python

```diff
@@ -0,0 +1,12 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal
+
+USER root
+RUN mkdir -p /app/ARO-RP
+WORKDIR /app/ARO-RP
+
+#we have multiple steps for copy so we can make use of caching
+COPY . /app/ARO-RP
+
+RUN microdnf update && \
+    microdnf install make python39 git && \
+    microdnf clean all
```

Comment:
does that mean that we will update all the packages before every run ? couldn't we build an image with those built in so we don't have to redownload and apply those everytime?